### ISSUE TITLE
Can't run the MacOS version

### ISSUE THREAD

**Issue by dbaldan** [Windblade-GR01/Ikemen-GO#318]
Hello guys, O Ilusionista here. Nice to meet you.
I've much experience in Mugen but I am a baby when comes to Ikemen, so sorry if I made some dumb question :)

I've tried to run the MacOS version but with no luck.
The two methods I've tried:

• Running the MacOS build - I just got a terminal screen with` "Ikemen_GO_MacOS ; exit;"` when I double click on _Ikemen_GO_MacOS_. My OSX version is 10.11 El Capitain - what is the minimum version allowed?

• Running the Windows version though Wine - I can run Mugen fine using Wine, but Ikemen does nothing to me
If I click on "Ikemen_GO_x86.exe", I just got an empty ikemen.log. If I click on "Ikemen_GO" nothing really happens.

Am I doing something wrong?

**Comment by Windblade-GR01** [comment]
The mimum version is 10.11l.
The builds on GitHub use the Mac OS 10.11 SDK.

I would say that is a problem with the build.... but it's weird that does not run even on WINE.
Did you download the "CoreOnly" zip?

**Comment by dbaldan** [comment]
Mine is 10.11. I've downloaded the "Ikemen_GO_v0.97.0.zip", not the Core only.

**Comment by Windblade-GR01** [comment]
Uh.
So I'm currently seeking to configure a MacOS VM.
Maybe something did go wrong with the compilation.

**Comment by Windblade-GR01** [comment]
O Ilusionista, I had tested running on a MacOS VM and it works.
Not sure what could be happening.

So, forgot to ask.
Do `ikemen.log` has anything on it?

**Comment by dbaldan** [comment]
The full report is:

```
panic: Main lua file 'external/script/main.lua' can not be found.

goroutine 1 [running, locked to thread]:
main.main()
	/ikemen/src/main.go:78 +0x73f
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...25 completed.

[Process completed]
```

**Comment by Windblade-GR01** [comment]
Uh? So the executable is working.
But for some reason it seems that is not loading the `external` folder.

This also explains why does not work even on WINE.
Did you run the .zip as/is or did you paste the executable somewhere else?

**Comment by dbaldan** [comment]
For sure I've extracted it first - OSx doesn't let you run files directly from .zip (at least, on my version).

First time, I've downloaded and extracted it directly - OSx creates a folder with the same name of the zip file "Ikemen_GO_v0.97.0".

Yesterday, I've downloaded it again to test and I had the same result. When I decided to rename the folder to just "ikemen" (so it won't have dots its name", it had at least executed and gave me the warning I've wrote above.

It's strange because the folder and the file is there - I can open the file normally.

Where `ikemen.log` is supposed to be? I can't see it.
